<launch>
    <node name="scan_values_handler" pkg="prrexamples" type="scan_values_handler.py" output="log"/>
    <node name="driver" pkg="prrexamples" type="driver.py" output="screen"/>
    <node name="pid" pkg="prrexamples" type="pid.py" output="screen"/>
</launch>